src/main/java/org/generationcp/middleware/domain/germplasm/GermplasmCodeNameBatchRequestDto.java

```diff
@@ -0,0 +1,54 @@
+package org.generationcp.middleware.domain.germplasm;
+
+import org.generationcp.middleware.pojos.germplasm.GermplasmNameSetting;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.List;
+
+@AutoProperty
+public class GermplasmCodeNameBatchRequestDto {
```

Comment:
Is this DTO specific to germplasm code name or can be it a generic germplasm name batch request dto?

Reply:
Yep this is specific to germplasm code name because we need to pass the germplasm name settings, and it only supports CODE1,2,3. Other names don't have that option.